### PR TITLE
Task/remove ac tool

### PR DESCRIPTION
**Pull request by vivekmalviya0104**
1. Refactoring the code 
2. Using now repoinit script to create users and groups
3. Adding rep:policy nodes, containing restrictions for dam path as content package
4. Changed the app path from /apps/valtech/aapm-examples to /apps/aapm-examples, because the repoint cfg.json triggers the slingRepositoryInitializer on 2nd level /apps/aapm-examples/osgiconfig/org.apache.sling.jcr.repoinit.RepositoryInitializer~aapm.cfg.json
5. Changed the ReadMe file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,14 +39,8 @@ There are 2 ways to deploy:
...
 - /content/dam/aapm-test
-- /apps/valtech/aapm
-- /apps/valtech/aapm-examples
-- /apps/netcentric/actool/config
-- /apps/netcentric/actool/config.dev
-- /apps/netcentric/actool/config.stage
-- /apps/netcentric/actool/config.prod
-- /apps/netcentric/actool/install
-- /etc/packages/Netcentric
+- /apps/aapm
```
Comment (gruberrolandvaltech) [inline README.md:42]: The tool is in /apps/valtech/aapm(-examples)

File: all/pom.xml [hunk 2]
```diff
@@ -55,7 +55,7 @@
                         <embedded>
                             <groupId>com.valtech.aapm</groupId>
                             <artifactId>aapm.core</artifactId>
-                            <target>/apps/valtech/aapm/install</target>
+                            <target>/apps/aapm/install</target>
```
Comment (gruberrolandvaltech) [inline all/pom.xml:58]: Please keep /apps/valtech

File: all/src/main/content/META-INF/vault/filter.xml [hunk 3]
```diff
@@ -15,5 +15,5 @@
   ~ limitations under the License.
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
 <workspaceFilter version="1.0">
-    <filter root="/apps/valtech/aapm"/>
```
Comment (gruberrolandvaltech) [inline all/src/main/content/META-INF/vault/filter.xml:18]: Keep the valtech

File: README.md (hunk 1 above)
Comment (gruberrolandvaltech) [inline README.md:42]: Keep valtech

File: examples.ui.apps/src/main/content/jcr_root/apps/aapm-examples/pbop/permissions/users/pbop-ace-user.yaml [hunk 4]
```diff
@@ -1,3 +1,6 @@
+# Deprecated as now using repoinit to create users and groups.
```
Comment (gruberrolandvaltech) [inline examples.ui.apps/src/main/content/jcr_root/apps/aapm-examples/pbop/permissions/users/pbop-ace-user.yaml:1]: No need for deprecation

File: examples.ui.config/src/main/content/META-INF/vault/filter.xml [hunk 5]
```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
-    <filter root="/apps/valtech/aapm-examples/osgiconfig"/>
+    <filter root="/apps/aapm-examples/osgiconfig"/>
```
Comment (gruberrolandvaltech) [inline examples.ui.config/src/main/content/META-INF/vault/filter.xml:3]: keep valtech

File: examples.ui.config/src/main/content/jcr_root/apps/aapm-examples/osgiconfig/config/org.apache.sling.commons.log.LogManager.factory.config~aapm.cfg.json [hunk 6]
```diff
@@ -2,7 +2,7 @@
   "org.apache.sling.commons.log.names": [
     "com.valtech.aapm"
   ],
-  "org.apache.sling.commons.log.level": "INFO",
+  "org.apache.sling.commons.log.level": "DEBUG",
```
Comment (gruberrolandvaltech) [inline examples.ui.config/src/main/content/jcr_root/apps/aapm-examples/osgiconfig/config/org.apache.sling.commons.log.LogManager.factory.config~aapm.cfg.json:5]: No debug logging by default, I think we can remove the whole config. There is no need for an extra log file for most users.

File: examples/pom.xml [hunk 7]
```diff
@@ -53,29 +53,23 @@
...
                             <type>zip</type>
-                            <target>/apps/valtech/aapm-examples/install</target>
+                            <target>/apps/aapm-examples/install</target>
                         </embedded>
                         <embedded>
                             <groupId>com.valtech.aapm</groupId>
                             <artifactId>examples.aapm.ui.content</artifactId>
                             <type>zip</type>
-                            <target>/apps/valtech/aapm-examples/install</target>
+                            <target>/apps/aapm-examples/install</target>
```
Comment (gruberrolandvaltech) [inline examples/pom.xml:66]: keep valtech

File: examples/src/main/content/META-INF/vault/filter.xml [hunk 8]
```diff
@@ -15,5 +15,5 @@
   ~ limitations under the License.
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
 <workspaceFilter version="1.0">
-    <filter root="/apps/valtech/aapm-examples"/>
+    <filter root="/apps/aapm-examples"/>
```
Comment (gruberrolandvaltech) [inline examples/src/main/content/META-INF/vault/filter.xml:18]: keep valtech

**Review by gruberrolandvaltech — CHANGES_REQUESTED** [review]

**Comment by vivekmalviya0104** [comment]
Added back the valtech folder

File: examples.ui.apps/pom.xml [hunk 9]
```diff
@@ -98,35 +98,12 @@
...
-                    </dependencies>
-                </configuration>
-            </plugin>
         </plugins>
     </build>
 
 
     <!-- ====================================================================== -->
-    <!-- D E P E N D E N C I E S                                                -->
+    <!-- D E P E Npob D E N C I E S                                                -->
```
Comment (gruberrolandvaltech) [inline examples.ui.apps/pom.xml:106]: pob?

**Review by gruberrolandvaltech — APPROVED** [review]

**Pull request merged**